Started by user [8mha:////4Dafb0UfgSpvFgtXEXdNGgm1IvxANvrx0SbXsRrSZ09OAAAAmB+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAz+EgYe/dLi1CL94oz8pMQ8AIJoyCbBAAAA[0mShoban Cheekuru
Building in workspace /Users/scheekur/.jenkins/workspace/devops-practice/sample-maven
No credentials specified
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/shobancs/spring-websample.git # timeout=10
Fetching upstream changes from https://github.com/shobancs/spring-websample.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/shobancs/spring-websample.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision bd322d8b1e7f8c4a3efedb28bad365445d808c88 (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f bd322d8b1e7f8c4a3efedb28bad365445d808c88 # timeout=10
Commit message: "Merge pull request #3 from shobancs/bala_dev"
 > git rev-list --no-walk bd322d8b1e7f8c4a3efedb28bad365445d808c88 # timeout=10
Parsing POMs
Discovered a new module org.springframework:gs-spring-boot gs-spring-boot
Modules changed, recalculating dependency graph
Established TCP socket on 58188
[initial] $ /bin/java -cp /Users/scheekur/.jenkins/plugins/maven-plugin/WEB-INF/lib/maven35-agent-1.12.jar:/usr/local/Cellar/maven/3.6.0/libexec/boot/plexus-classworlds-2.5.2.jar:/usr/local/Cellar/maven/3.6.0/libexec/conf/logging jenkins.maven3.agent.Maven35Main /usr/local/Cellar/maven/3.6.0/libexec/ /usr/local/apache-tomcat-9.0.14/webapps/jenkins/WEB-INF/lib/remoting-3.27.jar /Users/scheekur/.jenkins/plugins/maven-plugin/WEB-INF/lib/maven35-interceptor-1.12.jar /Users/scheekur/.jenkins/plugins/maven-plugin/WEB-INF/lib/maven3-interceptor-commons-1.12.jar 58188
ERROR: Failed to parse POMs
java.io.IOException: Cannot run program "/bin/java" (in directory "/Users/scheekur/.jenkins/workspace/devops-practice/sample-maven/initial"): error=2, No such file or directory
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at hudson.Proc$LocalProc.<init>(Proc.java:249)
	at hudson.Proc$LocalProc.<init>(Proc.java:218)
	at hudson.Launcher$LocalLauncher.launch(Launcher.java:936)
	at hudson.Launcher$ProcStarter.start(Launcher.java:455)
	at hudson.maven.AbstractMavenProcessFactory.newProcess(AbstractMavenProcessFactory.java:278)
	at hudson.maven.ProcessCache.get(ProcessCache.java:236)
	at hudson.maven.MavenModuleSetBuild$MavenModuleSetBuildExecution.doRun(MavenModuleSetBuild.java:804)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:504)
	at hudson.model.Run.execute(Run.java:1810)
	at hudson.maven.MavenModuleSetBuild.run(MavenModuleSetBuild.java:543)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)
Caused by: java.io.IOException: error=2, No such file or directory
	at java.lang.UNIXProcess.forkAndExec(Native Method)
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
	at java.lang.ProcessImpl.start(ProcessImpl.java:134)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 12 more
Finished: FAILURE
